File: Extras x RetosJS/5 | Cantidad de Palabras/5-Extra.js
/* 
    * Contar la cantidad de palabras en una frase
    * Completa la función contarPalabras que recibe una frase y devuelve la cantidad de palabras que contiene
*/

// ? Solución

const contarPalabras = (frase) => {
	let palabras = [];
	palabras = frase.split(' ');
	return palabras.length;
};

// Todo: Pruebas
console.log(contarPalabras('Hola Mundo ¿Como va todo?')) // Resultado: 5

File: Extras x RetosJS/5 | Cantidad de Palabras/5-Extra.py
# Contar la cantidad de palabras en una frase
# Completa la función contarPalabras que recibe una frase y devuelve la cantidad de palabras que contiene

def countWords(phrase):
    words = phrase.split(' ')
    return len(words)

print(countWords('Hola mundo ¿Como va todo?'))